codespell_lib/data/dictionary.txt

```diff
@@ -7444,28 +7475,36 @@ mkaing->making
 mke->make
 mkea->make
 mmaped->mapped
+mmaping->mmapping
```

Comment:
`mmapping` ? shouldn't this be `mapping` ?

Reply:
It's about memory mapping.

Anyway, removed.